provider/pkg/provider/provider.go

```diff
@@ -611,7 +611,10 @@ func (p *cfnProvider) Create(ctx context.Context, req *pulumirpc.CreateRequest)
 		cfType = spec.CfType
 
 		// Convert SDK inputs to CFN payload.
-		payload = schema.SdkToCfn(&spec, p.resourceMap.Types, inputs.MapRepl(nil, mapReplStripSecrets))
+		payload, err = schema.SdkToCfn(&spec, p.resourceMap.Types, inputs.MapRepl(nil, mapReplStripSecrets))
+		if err != nil {
+			return nil, fmt.Errorf("failed to convert SDK inputs to CFN: %w", err)
```

Comment:
Nit: Should we use `errors.Wrapf` instead? It seems like a slightly better API and we use it in other places.

Reply:
No, it looks like that API [is obsolete now](https://stackoverflow.com/a/61955455). It's not a huge deal to leave the existing usages for now, but I've been using the `%w` approach for all new code.

Reply:
TIL!